### PR TITLE
autopath for stripmap at univ. miami

### PR DESCRIPTION
**Pull request by bhuvankumaru**
Replacing this file into Mintpy/mintpy/defaults will temporarily/permanently solve importing stripmap products into MintPy.

### CODE REVIEW

File: minsar/additions/mintpy/auto_path.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+mintpy.load.lookupYFile    = ${PROJECT_DIR}/merged/geom_master/lat.rdr
+mintpy.load.lookupXFile    = ${PROJECT_DIR}/merged/geom_master/lon.rdr
+mintpy.load.incAngleFile   = ${PROJECT_DIR}/merged/geom_master/los.rdr
+mintpy.load.azAngleFile    = ${PROJECT_DIR}/merged/geom_master/los.rdr
+mintpy.load.shadowMaskFile = ${PROJECT_DIR}/merged/geom_master/shadowMask.rdr
+mintpy.load.bperpFile      = None
+'''
+iscestripmapAutoPath = '''##----------Default file path of ISCE-topsStack products
+mintpy.load.processor      = isce
+mintpy.load.metaFile       = ${PROJECT_DIR}/masterShelve/data.dat
```
Comment (mirzaees) [inline minsar/additions/mintpy/auto_path.py:44]: The path for metaFile is not correct

**Review by mirzaees — CHANGES_REQUESTED** [review]

File: minsar/additions/mintpy/auto_path.py (hunk 1 above)
Comment (bhuvankumaru) [reply minsar/additions/mintpy/auto_path.py:44]: I am making a copy of masterShelve in the project directory. SOmethings to be consistent with topsstack where the info is from ${PROJECT_DIR}/master/IW1.xml. Also the function mainly takes processor name and project name as inputs.

Comment (bhuvankumaru) [reply minsar/additions/mintpy/auto_path.py:44]: @mirzaees Is there a way to get the master date info into smallBaseline_wrapper. At that point, inputs dictionary has master date as 'None', so, I am directing it to the first date/folder in SLC directory. Can the masterdate info be updated into "inps"

Comment (mirzaees) [reply minsar/additions/mintpy/auto_path.py:44]: @bhuvankumaru I see, thank you
It is not really important to give it the master image as the masterShelve in all SLC folders are the same
smallbaselineWrapper.py can read the master from template file.
generally it is not a good idea to detach from the main repo very much, I have made a PR to mintpy, let's see if Yunjun does not approve, we can temporarily add this

**Pull request closed without merging**